pyproject.toml

```diff
@@ -59,14 +59,14 @@ vision = [
     "h5py>=3.10.0",
     "nibabel>=5.2.0",
     "opencv-python-headless>=4.9.0.80",
-    "timm>=0.9.12",
+    "timm @ git+https://github.com/huggingface/pytorch-image-models.git@main",
```

Comment:
Should we open an issue to set this back to a specific version tag once there is a new release out?

Reply:
sure yeah - maybe it will be caught from the bot as well 🤔